config.py.example

```diff
@@ -9,6 +9,7 @@ LANGUAGE = 'EN'  # ISO 639-1 codes EN, DE, FR, and ZH currently supported.
 MAP_START = (12.3456, 34.5678)
 MAP_END = (13.4567, 35.6789)
 GRID = (2, 2)  # row, column
+# DISABLE_MARKERS = [0, 1, 2]
```

Comment:
Or treat it as an optional config value, and do not require another config update:
`if (not hasattr(config, 'DISABLE_MARKERS') or  worker_no not in config.DISABLE_MARKERS):`
Is it bad approach?